### ISSUE TITLE
Installation instruction missing in *.zip and *.tar.gz archive

### ISSUE THREAD

**Issue by Isolator70** [ksnip/plugin-ocr#6]
The readme.md does not contain any useful installation instruction for users like me which do not instantly see which files needs which action.
Please add what has to be done in the Terminal - many thanks :-)

(As I'm here on Manjaro the provided packages do not match, also AUR does not contain it...)

**Comment by Moondev0607** [comment]
Agree, instructions unclear. Also building from [these](https://github.com/ksnip/kColorPicker#building-from-source) instructions seem to work, and installing the built files outputs a `/usr/local/lib/libksnip-plugin-ocr.so` which is not being recognized in the kSnip's extension list.
